__testfixtures__/prefer-type-aliases-arrow-function.input.js

```diff
@@ -0,0 +1,11 @@
+import PropTypes from "prop-types"
```

Comment:
One test is plenty, definitely don't need to duplicate every single test.

Reply:
Addressed